statsd/src/main/java/com/avast/metrics/statsd/StatsDMetricsMonitor.java

```diff
@@ -32,11 +32,11 @@
     protected final Map<String, ScheduledFuture<?>> gauges = new HashMap<>();
 
     public StatsDMetricsMonitor(String host, int port, boolean autoRegisterMetrics, String prefix, final Naming naming, final Duration gaugeSendPeriod, final ScheduledExecutorService scheduler, MetricsFilter metricsFilter) {
-        this.prefix = prefix;
+        this.prefix = sanitizePrefix(prefix);
```

Comment:
We often have prefix like `viruslab.scavenger.app1` so this sanitization doesn't make sense for us :(

Maybe the cleanest way is to introduce a new (case)class that would represent already sanitized metric name?